gradle/run.gradle

```diff
@@ -31,9 +31,14 @@ import org.opensearch.gradle.testclusters.RunTask
 
 apply plugin: 'opensearch.testclusters'
 
+def numNodes = findProperty('numNodes') as Integer ?: 1
+def numZones = findProperty('numZones') as Integer ?: 1
+
 testClusters {
   runTask {
     testDistribution = 'archive'
+    if (numZones > 1) numberOfZones = numZones
+    if (numNodes > 1) numberOfNodes = numNodes
```

Comment:
I don't know a lot about zones in this context, but there is already a numberOfNodes property that runs a two node cluster with gradle run, can we reuse that with added debug functionality? If not we should to document the differences.